interfaces/console/src/app/console/sites/[id]/page.tsx

```diff
@@ -251,131 +275,8 @@ const Page: React.FC<SiteDetailsProps> = ({ params }) => {
-    onCompleted: (res) => {
-      if (res.getComponentsByUserId) {
-        setComponentsList(res.getComponentsByUserId.components);
-      }
-    },
-    onError: (error) => {
-      setSnackbarMessage({
-        id: 'components-msg',
-        message: error.message,
-        type: 'error' as AlertColor,
-        show: true,
-      });
-    },
-  });
-
-  const { data: networks } = useGetNetworksQuery({
-    fetchPolicy: 'cache-and-network',
-    onError: (error) => {
-      setSnackbarMessage({
-        id: 'networks-msg',
-        message: error.message,
-        type: 'error' as AlertColor,
-        show: true,
-      });
-    },
-  });
-
-  const [getMetricStat] = useGetMetricsStatLazyQuery({
-    client: subscriptionClient,
-    fetchPolicy: 'network-only',
-    onCompleted: (data) => {
-      if (data.getMetricsStat.metrics.length > 0) {
-        data.getMetricsStat.metrics.forEach((m) => {
-          if (m.type === 'unit_uptime') {
-            setNodeUptime(m.value);
-          }
-        });
-
-        const sKey = `stat-${user.orgName}-${user.id}-${Stats_Type.AllNode}-${statVar?.data.from ?? 0}`;
-        MetricStatSubscription({
-          key: sKey,
-          nodeId: id,
-          userId: user.id,
-          url: env.METRIC_URL,
-          orgName: user.orgName,
-          type: Stats_Type.AllNode,
-          from: statVar?.data.from ?? 0,
-        });
-        PubSub.subscribe(sKey, handleStatSubscription);
-      }
-    },
-  });
-
-  useEffect(() => {
-    const to = getUnixTime();
-    const from = to - STAT_STEP_29;
-    if (!id) {
-      setSnackbarMessage({
-        id: 'node-not-found-msg',
-        message: 'Node not found.',
-        type: 'error',
-        show: true,
-      });
-      router.back();
-    } else if (id) {
-      const to = getUnixTime();
-      const from = to - STAT_STEP_29;
-      getMetricStat({
-        variables: {
-          data: {
-            to: to,
-            nodeId: id,
-            from: from,
-            userId: user.id,
-            step: STAT_STEP_29,
-            orgName: user.orgName,
-            withSubscription: true,
-            type: Stats_Type.AllNode,
-          },
-        },
-      });
-    }
-    return () => {
-      const sKey = `stat-${user.orgName}-${user.id}-${Stats_Type.AllNode}-${from ?? 0}`;
-      PubSub.unsubscribe(sKey);
-    };
-  }, []);
   const [getSite, { loading: getSiteLoading }] = useGetSiteLazyQuery({
     onCompleted: (res) => {
-      setSite({
-        power: res.getSite.powerId,
-        siteName: res.getSite.name,
-        switch: res.getSite.switchId,
-        access: res.getSite.accessId,
-        address: res.getSite.location,
-        latitude: res.getSite.latitude,
-        network: res.getSite.networkId,
-        spectrum: res.getSite.spectrumId,
-        backhaul: res.getSite.backhaulId,
-        longitude: res.getSite.longitude,
-      });
       setActiveSite({
         id: res.getSite.id,
```

Comment:
Cant we use the same response state rather maintaining another state?

Reply:
addressed.